clients/src/main/java/org/apache/kafka/common/network/SslTransportLayer.java

```diff
@@ -168,12 +168,11 @@ public void close() {
             try {
                 socketChannel.socket().close();
                 socketChannel.close();
-            } catch (IOException e) {
-                log.warn("Failed to close SSL socket channel: " + e);
```

Comment:
Looking at this some more, one downside is that we will now log these as errors and with the stacktrace. I remember you originally suggested that it was noisy to include the stacktrace here. What are your thoughts on that?